collections/proxy/proxy_golden_path/happy_path/discovery.json

```diff
@@ -0,0 +1,326 @@
+{
+    "name": "Party Lookup Happy Path",
```

Comment:
I'd suggest adding a check of receiving  proxyList from Central ledger as a first test (to make sure we have proper setup)